hbot_install.md

```
@@ -63,28 +48,30 @@ sudo ufw allow from 127.0.0.1 to 127.0.0.1 port 15888 proto tcp
 sudo ufw enable
 ```
 
-> **Note**: Replace `10.0.0.10` with the IP address of the system or network allowed to access your server via SSH. Use `0.0.0.0/0` to allow all IPs, but this is less secure.
+> **Note**: Replace `10.0.0.10` with the IP address of the system or network allowed to access your server via SSH. 
+> 
+> Also, see `/src/templates/server.yml` for `ipWhitelist: []` option. 
 
-Enable logging
+**Enable logging**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)